src/mock/ytelse/statiskSykepengerMock.ts

```diff
@@ -0,0 +1,60 @@
+import { Sykepenger } from '../../models/ytelse/sykepenger';
+import { statiskKommendeUtbetaling } from './statiskForeldrepengeMock';
+
+const arbeidsforhold = {
```

Comment:
Kunne vi gjort denne dataen VELDIG opplagt at den er rent oppspinn? Både navnene og datoene her er ikke egentlig så opplagte. Og for å unngå det problemet mange team har hatt (inkludert oss, stakkars Eigil) så er det finfint om ting er idiotsikkert at det er testdata, spesielt når Eline Moe ikke er stort med enn Aremark Testesen i testene :)

Reply:
Bra tips det, skal fikse!